index.html

```diff
@@ -7,5 +7,14 @@
 </head>
 <body>
```

Comment:
Je had voor het gemak en tijdwinst beter 1 class voor alle items kunnen maken en ieder los item een id.
Hiermee defineer je in de class alle zaken die ze allemaal gelijk hebben(display, justify-content, align-items) en hoef je alleen je gridarea maar vast te zetten per id en eventuele eigenschappen die alleen op die area van toepassing zijn.